lib/datadog/core/configuration/settings.rb

```diff
@@ -136,11 +141,12 @@ def initialize(*_)
             # If `nil`, defaults to logging startup logs when `ddtrace` detects that the application
             # is *not* running in a development environment.
             #
-            # @default `DD_TRACE_STARTUP_LOGS` environment variable, otherwise `nil`
-            # @return [Boolean,nil]
+            # @default `DD_TRACE_STARTUP_LOGS` environment variable, otherwise `false`
+            # @return [Boolean]
```

Comment:
See my comment on environment_logger; if you end up agreeing, this change to remove the `nil` probably needs reverting.

Reply:
Done https://github.com/DataDog/dd-trace-rb/pull/2988/commits/165138d90e99ad6cadc95de464bac50216a4bb69